packages/start-server-core/vite.config.ts

```
@@ -4,14 +4,23 @@ import packageJson from './package.json'
 // this needs to be imported from the actual file instead of from 'index.tsx'
 // so we don't trigger the import of a `?script-string` import before the minifyScriptPlugin is setup
 import { VIRTUAL_MODULES } from './src/virtual-modules'
-
+import path from 'path'
```

ESLint

[error] Expected 1 empty line after import statement not followed by another import.

(import/newline-after-import)